packages/volto-slate/news/7179.bugfix

```diff
@@ -0,0 +1 @@
+Correct README, replace tbody with tfoot, serve tabIndex prop with number. @silviubogan
```

Comment:
```suggestion
Correct `README.md`. Set the correct value of the constant `TFOOT` to `tfoot`, instead of `tbody`. Serve `tabIndex` prop with a number instead of a string. @silviubogan
```